setup.py

```diff
@@ -209,6 +209,33 @@ def initialize_options(self):
     def finalize_options(self):
         pass
 
+    def get_language_checkers(self):
+        from translate.lang.factory import getlanguage
+
+        def get_languages_with_module():
```

Comment:
I think move this whole function into translate.lang.factory as there is already a getlanguage.  Just create a getlanguages function that returns classes for all of these.